src/browser/TestBrowser.ts

```diff
@@ -80,7 +80,15 @@ export default class TestBrowser {
       });
     }
 
-    return Result.fromPage(this.page, story);
+    const axeResults = await AxePage.analyze(
+      this.page,
+      story.disabledRules,
+      story.runOptions,
+      story.context,
+      story.config,
+    );
```

Comment:
As noted elsewhere, I moved this code here. That way the Result object doesn't need to contain any Playwright stuff.